aip/client-libraries/4236.md

```diff
@@ -0,0 +1,36 @@
+---
+id: 4236
+state: approved
+created: 2024-05-16
+---
+
+# Version-aware clients
+
+APIs may choose to annotate services with [google.api.api_version]. If 
+[google.api.api_version] is specified, version-aware clients **must** include
+the value of [google.api.api_version] in the request to the API. This allows
+services to abide by the schema and behavior of the service at the time that
+this API version was deployed. The format of the API version must be treated
+as opaque by clients. Services may use a format with an apparent structure,
+but clients must not rely on this to determine components within an API version,
+or attempt to construct other valid API versions.
```

Comment:
```suggestion
as opaque by clients. Clients **must not** rely on any apparent format in the value for
any reason.
```

Let's just tighten up the guidance, be a bit more explicit, and explain things more in `Rationale`.